### DISCUSSION TITLE
Is there anything so we can need to use a password to access the website

### DISCUSSION THREAD

**Discussion by pumpkin2557** [plan-player-analytics/Plan#2511] [Q&A] [answered]
I don't want normal users to access plan website. Anyway to set a password which you need to enter to access the website?

**Comment by AuroraLS3** [comment, marked as answer]
https://github.com/plan-player-analytics/Plan/wiki/SSL-Certificate-%28HTTPS%29-Set-Up

**Reply by pumpkin2557** [reply]
Thanks alot! <3
